compiler/stable_mir/src/lib.rs

```diff
@@ -69,6 +69,7 @@ pub type ImplTraitDecls = Vec<ImplDef>;
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub enum CompilerError<T> {
     /// Internal compiler error (I.e.: Compiler crashed).
+    #[allow(non_camel_case_types)]
     ICE,
```

Comment:
```suggestion
    Ice,
```
`Ice` is fine in this context.